ansible/roles/tasks-systemd/tasks/main.yml

```diff
@@ -40,6 +40,54 @@
       maxsockets=3
       cafile=/run/secrets/tasks/npm-registry.crt
 
+- name: Create job-runner configuration
+  copy:
+    dest: /etc/job-runner.toml
+    content: |
+      [logs]
+      driver='s3'
+
+      [forge.github]
+      token = [{file="/run/secrets/webhook/.config--github-token"}]
```

Comment:
(feel free to say no) can you get us an extra `--volume` for /var/secrets/github-token and use it from that location?